code/model/Post.php

```diff
@@ -100,48 +100,48 @@ function canDelete($member = null) {
 			return $this->Thread()->canModerate($member);
 		}
 	}
-	
+
 	/**
 	 * Check if user can add new posts - hook up into canPost.
 	 */
-	function canCreate($member = null) {
+	public function canCreate($member = null) {
 		if(!$member) $member = Member::currentUser();
 		return $this->Thread()->canPost($member);
 	}
-	
+
 	/**
 	 * Returns the absolute url rather then relative. Used in Post RSS Feed
 	 *
 	 * @return String
 	 */
-	function AbsoluteLink() {
+	public function AbsoluteLink() {
 		return Director::absoluteURL($this->Link());
 	}
 
 	/**
 	 * Return the title of the post. Because we don't have to have the title
 	 * on individual posts check with the topic
-	 * 
+	 *
 	 * @return String
 	 */
-	function getTitle() {
-		return ($this->isFirstPost()) ? $this->Thread()->Title : sprintf(_t('Post.RESPONSE',"Re: %s",'Post Subject Prefix'),$this->Thread()->Title);
+	public function getTitle() {
+		return ($this->IsFirstPost()) ? $this->Thread()->Title : sprintf(_t('Post.RESPONSE',"Re: %s",'Post Subject Prefix'),$this->Thread()->Title);
 	}
 
 	/**
 	 * Return the last edited date, if it's different from created
 	 */
-	function getUpdated() {
+	public function getUpdated() {
 		if($this->LastEdited != $this->Created) return $this->LastEdited;
 	}
-	
+
 	/**
 	 * Is this post the first post in the thread. Check if their is a post with an ID less
 	 * than the one of this post in the same thread
 	 *
 	 * @return bool
 	 */
-	public function isFirstPost() {
+	public function IsFirstPost() {
```

Comment:
I'm not sure what our convention is on these kinds of functions, but I do usually use lowercase `is`...